Stripe/PublicHeaders/STPPaymentContext.h

```diff
@@ -372,7 +372,20 @@ didCreatePaymentResult:(STPPaymentResult *)paymentResult
             completion:(STPErrorBlock)completion;
 
 /**
- This is invoked by an `STPPaymentContext` when it is finished. This will be called after the payment is done and all necessary UI has been dismissed. You should inspect the returned `status` and behave appropriately. For example: if it's `STPPaymentStatusSuccess`, show the user a receipt. If it's `STPPaymentStatusError`, inform the user of the error. If it's `STPPaymentStatusUserCanceled`, do nothing.
+ This method is only called when the following happens:
+    1. The user uses apple pay
+    2. A token for stripe is created and we call the delegate method with the parameter named didCreatePaymentResult is called
+    2. The user taps Apple pay's cancel button AFTER we already called the delegate method with didCreatePaymentResult
+ When this is called the user will expect you to cancel the payment but that may be difficult if you already sent the stripe token to your server. You could try
+ to give the user a refund or just tell them that they can't cancel.
+
+ Note that if the user cancel BEFORE the delegate method with didCreatePaymentResult is called then didFinish will be called and this method will not be called.
+ */
+- (void)paymentContext:(STPPaymentContext *)paymentContext
```

Comment:
this seems like a pretty reasonable method name to me. Let's wait for a thumbs up from @danj-stripe as well though